libs/dmlf/tests/execution/basic_engine_tests.cpp

```diff
@@ -1905,13 +1873,30 @@ TEST(DISABLED_BasicVmEngineDmlfTests, DisabledArrayArrayOpTests)
 {
   RunArrayTest("doFloat32", std::vector<float>{1.0f, 2.0f, 3.0f, 4.0f});
 
-  RunArrayTest("doFixed32",
```

Comment:
Oh, because they are working! I have this disabled test to show which parameters/return values don't work when using arrays. I had fixed the issue but incorrectly hadn't deleted them from here. They are in a test a little bit up.